packages/xstate-react/src/createActorContext.tsx

```diff
@@ -15,24 +15,31 @@ export function createActorContext<TMachine extends AnyStateMachine>(
     selector: (snapshot: EmittedFrom<TMachine>) => T,
     compare?: (a: T, b: T) => boolean
   ) => T;
-  useActorRef: () => ActorRefFrom<TMachine>;
+  useContext: () => ActorRefFrom<TMachine>;
   Provider: (props: {
     children: React.ReactNode;
+    value?: TMachine | ActorRefFrom<TMachine>;
   }) => React.ReactElement<any, any>;
 } {
   const ReactContext = React.createContext<ActorRefFrom<TMachine> | null>(null);
 
   const OriginalProvider = ReactContext.Provider;
 
-  function Provider({ children }: { children: React.ReactNode }) {
-    const actor = useInterpret(machine) as ActorRefFrom<TMachine>;
+  function Provider({
+    children,
+    value = machine
```

Comment:
When it comes to the barebones React context `value` is both what you provide and what you consume. We break that parity here because we provide an actor ref under the hood but the `value` here is the machine. 

I'd probably name this prop a `machine` for this reason.

Reply:
OTOH, we already "break" the parity with the barebones React context as the "initial value" used to create this is different from what we end up providing.

But even considering this - I feel like a more descriptive prop name (`machine`) is better. React has to use a generic `value` because their whole signature is super generic and has to accept just any value.